plugins.go

```diff
@@ -109,7 +109,7 @@ func ListPlugins() map[string][]string {
 
 	// cluster plugins in registration order
 	for name := range clusterProviders {
-		p["clustering"] = append(p["clsutering"], name)
+		p["clustering"] = append(p["clustering"], name)
```

Comment:
This might be intentional, if it is, there should be a comment explaining it.

Reply:
Ha, wow, this is a bug! Thanks

Reply:
Yeah, bugs happen, and spell checkers are fairly good at finding a set of them. For any moderately sized project one or two are to be expected. Glad to be of service.